src/main/resources/init_db.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE bookstore;
+USE bookstore;
+CREATE TABLE books
+(
+    id    INT  AUTO_INCREMENT PRIMARY KEY,
```

Comment:
And yet another point from common mistakes 😞 
```suggestion
    id    BIGINT  AUTO_INCREMENT PRIMARY KEY,
```